monorepo/website/src/pages/docs/how-to/add-remove-user-group.mdx

```diff
@@ -1,20 +1,23 @@
 ---
 layout: ../../../layouts/DocLayout.astro
 title: "Adding or removing a user group"
-order: 5
+order: 21
 ---
 
-Users cannot add themselves to groups; they must ask someone within a group to add them. When making this request, please be sure to supply your username exactly as it appears within your user account, as this is what they will use to add you.
+Users cannot add themselves to groups; they must ask someone within a group to add them.
+When making this request, please be sure to supply your username exactly as it appears within your user account, as this is what they will use to add you.
 
 You can then ask them to follow the below steps to add you to their group.
 
-To add a user to your group:
+## Adding a user to your group

```

Comment:
```suggestion
## Adding a user to a group you are a member of

```

I'm not sure about this change, the wording predates this PR but thinking about it: "your group" reads more nicely but is a bit imprecise. It might give the wrong idea that there are owners of groups and that everyone has a group or belongs to one group, when in fact all members can add anyone to any groups they belong to.